health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/components/RoleTableComposer.js

```
@@ -39,16 +41,23 @@ function RoleTableComposer() {
       enabled: false,
       select: (data) => {
         const resp = data?.Employees?.map((item, index) => {
+          let selectedHierarchy = state?.boundaryHierarchy?.find(
```

Biome

[error] This let declares a variable that is only assigned once.

'selectedHierarchy' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)